### ISSUE TITLE
only display an idea's vote count once voting stage is complete

### ISSUE THREAD

**Issue by Trevoke** [stride-nyc/remote_retro#295]
As a facilitator,
I want the number of votes on each item to be hidden until voting is done
So that team members aren't swayed by a high or low number and vote according to their preference only.

__Acceptance Criteria:__

- [ ] Vote count is not revealed to participants until all votes are in

**Comment by vanderhoop** [comment]
Hey @Trevoke, gonna close this one as a duplicate of #277! Take solace: we *are* prioritizing this!
